src/core/colorspace.js

```diff
@@ -396,7 +396,8 @@ class ColorSpace {
               }
             }
           }
-          throw new FormatError(`Unrecognized ColorSpace: ${cs.name}`);
```

Comment:
Here, and elsewhere below, can we keep these messages as `warn()` or `info()`, since that may be helpful when debugging?